From 013311e94d4fcebe3f601c2e434acd53254982cc Mon Sep 17 00:00:00 2001
From: Tal Yacobi <tyacobi@habana.ai>
Date: Wed, 11 Jun 2025 12:23:47 +0300
Subject: [PATCH] [SW-232024] gaudi3: pldm: rename spi stamp variable

The spi component comparison stamp needs to include
the svn value, which means it will be different than the other
fw loader components.

Rename the variable so it will be possible to set
the value separately in the automation scripts.

Change-Id: I1bc3fd77b043a8e3c4fb83f2798b49ab07e533d2
Signed-off-by: Tal Yacobi <tyacobi@habana.ai>
---

diff --git a/board/habanalabs/gaudi3/create-pldm-fwup-package-main.sh b/board/habanalabs/gaudi3/create-pldm-fwup-package-main.sh
index 99187ed..efee54c 100755
--- a/board/habanalabs/gaudi3/create-pldm-fwup-package-main.sh
+++ b/board/habanalabs/gaudi3/create-pldm-fwup-package-main.sh
@@ -45,7 +45,7 @@
                              --component $((COMPONENT_CLASSIFICATION_FIRMWARE)) $((COMPONENT_ID_FW_LOADER_1))   \
                                 "${AGENT_FW_LOADER_FIT_VERSION}" "${AGENT_FW_LOADER_FIT_VERSION_STAMP}"         \
                              --component $((COMPONENT_CLASSIFICATION_FIRMWARE)) $((COMPONENT_ID_SPI))           \
-                                "${AGENT_FW_LOADER_FIT_VERSION}" "${AGENT_FW_LOADER_FIT_VERSION_STAMP}"         \
+                                "${AGENT_FW_LOADER_FIT_VERSION}" "${SPI_FW_LOADER_FIT_VERSION_STAMP}"           \
                              --component $((COMPONENT_CLASSIFICATION_FIRMWARE)) $((COMPONENT_ID_FW_LOADER_2))   \
                                 "${AGENT_FW_LOADER_FIT_VERSION}" "${AGENT_FW_LOADER_FIT_VERSION_STAMP}"         \
                              --component $((COMPONENT_CLASSIFICATION_FIRMWARE)) $((COMPONENT_ID_EROM))          \
